samza-core/src/main/java/org/apache/samza/system/StandaloneExecutionEnvironment.java

```diff
@@ -28,7 +28,11 @@
 /**
  * This class implements the {@link ExecutionEnvironment} that runs the applications in standalone environment
  */
-public class StandaloneExecutionEnvironment implements ExecutionEnvironment {
+public class StandaloneExecutionEnvironment extends AbstractExecutionEnvironment {
```

Comment:
cc @xinyuiscool since this should probably be done while renaming to ApplicationRunner